### PR TITLE
faq: added .zenodo.json content

### PR DESCRIPTION
**Pull request by ctrl-alt-soneca**

### CODE REVIEW

**Comment by ctrl-alt-soneca** [comment]
Captura de ecrã 2021-06-28, às 17 41 50

File: content/help/contents.lr [hunk 1]
```diff
@@ -689,6 +689,18 @@ The project partners include:
 - CERN (Zenodo)
 ----
 category: citations-col2
+#### qaitem ####
+question: How to make metadata changes on my Github release?
```
Comment (slint) [inline content/help/contents.lr:693]:
```suggestion
question: How to change the default metadata for my Github releases?
```

File: content/help/contents.lr [hunk 2]
```diff
@@ -689,6 +689,18 @@ The project partners include:
...
+----
+answer: Zenodo's `.zenodo.json` file complies with the Zenodo metadata schema and should be used to update the metadata from the Github releases. For more information on how to proceed, you can take a look at our [developer docs](https://developers.zenodo.org/#add-metadata-to-your-github-repository-release).
```
Comment (slint) [inline content/help/contents.lr:695]:
```suggestion
answer: You can include metadata in a `.zenodo.json` file at the root of your GitHub repository. This file's contents should be in line with the Zenodo deposit metadata schema and will be used to update the metadata from future GitHub releases. For more information on how to create a `.zenodo.json` file, you can have a look at our [GitHub integration documenation](https://developers.zenodo.org/#github).
```

File: content/help/contents.lr [hunk 3]
```diff
@@ -689,6 +689,18 @@ The project partners include:
...
+----
+category: github
+#### qaitem ####
+question: What about other metadata schemas?
```
Comment (slint) [inline content/help/contents.lr:699]:
```suggestion
question: Do you support other software metadata files, like `CITATION.cff` or `codemeta.json`?
```

File: content/help/contents.lr [hunk 4]
```diff
@@ -689,6 +689,18 @@ The project partners include:
...
+----
+answer:Part of our plans and ongoing work is to integrate also other software metadata schemas such as `citation.cff` and `codemeta.json`. We are not sure yet exactly when or in what form all these files will be integrated with each other which is something we will decide based on the user and community feedback.
```
Comment (slint) [inline content/help/contents.lr:701]:
```suggestion
answer: Part of our plans and ongoing work is to integrate also other software metadata schemas such as the [Citation File Format (CFF)](https://citation-file-format.github.io/) and [CodeMeta](https://codemeta.github.io/). We are working closely with our users and scholarly software and citation communities to figure out exactly when, in what form, and how all these different metadata schemas and their files will be integrated into Zenodo.
```

**Review by slint — COMMENTED** [review]
Some wording/sentences and it should be good.

File: content/help/contents.lr (hunk 4 above)
Comment (ctrl-alt-soneca) [reply content/help/contents.lr]: Thank you for the quick review :)

**Pull request merged**